src/shared_modules/dbsync/src/sqlite/sqlite_dbengine.cpp

```diff
@@ -833,8 +833,7 @@ bool SQLiteDBEngine::getPKListLeftOnly(const std::string& t1,
 }
 
 std::string SQLiteDBEngine::buildDeleteBulkDataSqlQuery(const std::string& table,
-                                                        const std::vector<std::string>& primaryKeyList,
-                                                        const nlohmann::json& jsData)
+                                                        const std::vector<std::string>& primaryKeyList)
```

Comment:
is this right? I remember we needed to add this in order to support the delete without all the primary keys.